packages/cli/core/README.md

```diff
@@ -1067,6 +1067,36 @@ FLAGS
 
 DESCRIPTION
   Deploy your search application to the Coveo infrastructure.
+  The target `config` if present, must contain the following parameters:
```

Comment:
Hope it doesn't delete it again https://github.com/coveo/cli/commit/65e57c953aca07c47ac36267752e363ca4661ada

Reply:
It will.
You need to use `summary` for the brief tldr of your command, and `description` for the lengthy doc
https://oclif.io/docs/commands

Reply:
ahhh!